pontifical-gregorian-university.csl

```diff
@@ -12,22 +12,69 @@
     <category citation-format="note"/>
     <category field="theology"/>
     <summary>The Pontifical Gregorian University (Pontificia Università Gregoriana) style</summary>
-    <updated>2016-05-30T09:00:00+00:00</updated>
+    <updated>2016-06-08T08:26:40+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale>
     <style-options punctuation-in-quote="false"/>
     <terms>
       <term name="et-al"> &#8211; al.</term>
       <term name="ibid">Ibid.</term>
-      <term name="accessed">accesso</term>
+      <term name="in">in</term>
+      <term name="accessed">accesso:</term>
+      <term name="editor">
+        <single>ed.,</single>
+        <multiple>ed.,</multiple>
+      </term>
```

Comment:
> `<single>ed.,</single>`

It's generally better to keep separating punctuation (which I assume this comma is) out of the term definitions, and use a prefix/suffix or (even better yet) a delimiter instead.